doc/sg/c_cpp_coding_style.md

```diff
@@ -245,7 +245,7 @@ This feature is fairly mature in both compilers, though it varies from the C11 v
   - It can only be used with structs and unions, not arrays.
   - Members must be initialized in declaration order.
 
-Because it is especially useful with types declared for C, we allow designatued initializers whenever the type is a plain-old-data type, and:
+Because it is especially useful with types declared for C, we allow designated initializers whenever the type is a plain-old-data type, and:
```

Comment:
nit: **As** it is especially useful with types declared for C, we allow designated initializers whenever the type is a plain-old-data type, and: